.github/workflows/unit-tests.yaml

```diff
@@ -26,22 +26,5 @@ jobs:
           go install gotest.tools/gotestsum@latest
           go get -u github.com/proglottis/gpgme
 
-      # used to deploy the database container during unit testing
-      - name: Install podman 4
-        run: |
-          sudo mkdir -p /etc/apt/keyrings
-          curl -fsSL "https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/Release.key" \
-            | gpg --dearmor \
-            | sudo tee /etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg > /dev/null
-          echo \
-            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg]\
-              https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/ /" \
-            | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list > /dev/null
-          sudo apt-get update -qq
-          sudo apt-get -qq -y install podman
-
-      - name: Install podman-compose
-        run: pip3 install podman-compose
-
       - name: Running Unit tests
         run: make unit-test
```

Comment:
This now automatically creates a cluster with only the DB and the admin settings.